Python/Product/TestAdapter/visualstudio_py_testlauncher.py

```diff
@@ -101,23 +102,9 @@ def tell(self):
     def seek(self, pos, whence = 0):
         return self.buffer.seek(pos, whence)
 
-class _IpcChannel(object):
+class _IpcChannel(_vsipc.SocketIO, _vsipc.IpcChannel):
     def __init__(self, socket):
-        self.socket = socket
-        self.seq = 0
-        self.lock = thread.allocate_lock()
-
-    def close(self):
-        self.socket.close()
-
-    def send_event(self, name, **args):
-        with self.lock:
-            body = {'type': 'event', 'seq': self.seq, 'event':name, 'body':args}
-            self.seq += 1
-            content = json.dumps(body).encode('utf-8')
-            headers = ('Content-Length: %d\r\n\r\n' % (len(content), )).encode('ascii')
-            self.socket.send(headers)
-            self.socket.send(content)
+        super(_IpcChannel, self).__init__(socket=socket)
```

Comment:
I was debating this. Base class requires it to be pass by name.  I could change the caller to pass it by name instead of delete the init.  I think I'll do that.